src/aces/configs/aces.yaml

```diff
@@ -12,6 +12,9 @@ config_path: ${cohort_dir}/${cohort_name}.yaml
 # Path to store the output file. The `${data._prefix}` addition allows us to add shard specific prefixes in a
 # sharded data mode.
 output_filepath: ${cohort_dir}/${cohort_name}${data._prefix}.parquet
+# Optional path to store the output file with the raw window data.
+window_stats_dir: null
+window_stats_filename: ${window_stats_dir}/${cohort_name}${data._prefix}.parquet
```

Comment:
call this `_filepath` for consistency with `output_filepath`. in general, I use `filename` / `fn` for an arg that is just a file's name (e.g., does not contain the full path, though it can contain more than just the last part in some cases), `*dir` for the root directory of something (e.g., the inverse of `filename`), and `filepath` for the full path from root (Which this is). Though that is just my convention, here you should use it because we are already using that for `output`.